news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – April 2020 edition"
+subtitle: "Where you'll learn that PrestaShop is fully available in Danish"
+date:  2020-05-14 09:10:11
```

Comment:
Usually, Crowdin monthly are published on a Wednesday.
What do you think about next one, 20th of May 2020?

```suggestion
date:  2020-05-20 10:00:00
```

Reply:
Sounds good to me!